.eslintrc.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    'jest/globals': true
+  },
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
```

Comment:
I propose that we add prettier recommend rules
https://prettier.io/docs/en/integrating-with-linters.html

Reply:
`plugin:prettier/recommended` added